### ISSUE TITLE
Adding capability NET_ADMIN does not affect seccomp profile

### ISSUE THREAD

**Issue by nakedible-p** [moby/moby#33651]
**Description**

Containers with NET_ADMIN capability are not allowed to access pfkey subsystem for running the racoon IPsec daemon. Running the same container with `--security-opt seccomp:unconfined` allows this, so the problem is likely that the generated seccomp profile is missing calls that should be allowed. Indeed, looking at [the profile](../blob/master/profiles/seccomp/seccomp_default.go), there is no mention of NET_ADMIN.

**Steps to reproduce the issue:**
1. Create image with racoon daemon
2. Start container with `--cap-add NET_ADMIN`

**Describe the results you received:**

Racoon exits immediately, with the following error message in logs:

```
pfkey_open: Operation not permitted
```

**Describe the results you expected:**

Racoon should run normally, as it does with docker version 1.12.0, at least.

**Additional information you deem important (e.g. issue happens only occasionally):**

Adding `--security-opt seccomp:unconfined` to the command line makes the issue go away.

**Output of `docker version`:**

```
Client:
 Version:      17.03.1-ce
 API version:  1.27
 Go version:   go1.7.5
 Git commit:   7392c3b/17.03.1-ce
 Built:        Tue May 30 17:59:44 2017
 OS/Arch:      linux/amd64

Server:
 Version:      17.03.1-ce
 API version:  1.27 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   7392c3b/17.03.1-ce
 Built:        Tue May 30 17:59:44 2017
 OS/Arch:      linux/amd64
 Experimental: false
```

**Output of `docker info`:**

```
Client:
 Version:      17.03.1-ce
 API version:  1.27
 Go version:   go1.7.5
 Git commit:   7392c3b/17.03.1-ce
 Built:        Tue May 30 17:59:44 2017
 OS/Arch:      linux/amd64

Server:
 Version:      17.03.1-ce
 API version:  1.27 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   7392c3b/17.03.1-ce
 Built:        Tue May 30 17:59:44 2017
 OS/Arch:      linux/amd64
 Experimental: false
[naked@mpp-vpn1--i-079cd853f582e2b73 ~]$ sudo docker info
Containers: 2
 Running: 2
 Paused: 0
 Stopped: 0
Images: 2
Server Version: 17.03.1-ce
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins: 
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version:  (expected: 4ab9917febca54791c5f071a9d1f404867857fcc)
runc version: N/A (expected: 54296cf40ad8143b62dbcaa1d90e520a2136ddfe)
init version: N/A (expected: 949e6facb77383876aeff8a6944dde66b3089574)
Security Options:
 seccomp
  Profile: default
Kernel Version: 4.9.27-14.31.amzn1.x86_64
Operating System: Amazon Linux AMI 2017.03
OSType: linux
Architecture: x86_64
CPUs: 1
Total Memory: 993.5 MiB
Name: mpp-vpn1--i-079cd853f582e2b73
ID: EREJ:ZSH2:AJ3S:X3OO:5HO2:LHBO:XQZF:PJEJ:74OJ:T3F4:3WL7:U53V
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

Running in AWS.

**Comment by nakedible-p** [comment]
Probably related to #22554. Possibly a duplicate of #32847.

**Comment by thaJeztah** [comment]
Ping @justincormack PTAL. Not sure if this is by design, or if we want to have seccomp follow `NET_ADMIN`

**Comment by thaJeztah** [comment]
@nakedible-p Is this still an issue on the current release? I see https://github.com/moby/moby/issues/32847 was closed and fixed in Docker 17.06 and up. Your issue doesn't have exact steps to reproduce, so I couldn't test if it was fixed or not 😅

**Comment by thaJeztah** [comment]
closing as this went stale
